igvc_perception/src/elevation_map_height_node/elevation_map_height_node.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_ELEVATIONMAPHEIGHTNODE_H
+#define SRC_ELEVATIONMAPHEIGHTNODE_H
```

Comment:
Do you need to have this import when you already have the line above?

Reply:
These lines are there to ensure that `elevation_map_height_node.h` is only initialized once